### PR TITLE
Request Uri 

### PR DESCRIPTION
**Pull request by Elyniss**

### CODE REVIEW

**Review by IljaDaderko — APPROVED** [review]

File: sign/pairing-uri.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Schema ([EIP-1328](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1328.md))
 
-    uri         = "wc" ":" topic [ "@" version ][ "?" parameters ]
+    uri         = "wc" ":" "sign" "-" topic [ "@" version ][ "?" parameters ]
```
Comment (jakubuid) [inline sign/pairing-uri.md:5]: This change breaks backward compatibility in `sign`. We cannot be sure that all Dapps will use the new sdk in the same time.

We should either handle `sign` URI with prefix and without prefix on wallets side or just don't add such prefix and assume that when there is no prefix we consume `sign` sdk.

Comment (xzilja) [reply sign/pairing-uri.md:5]: what if this becomes uri param instead? i.e. `?sdk=sign` if there is no param we assume it is sign

Comment (Elyniss) [reply sign/pairing-uri.md:5]: That's why I've added Backwards Compatibility section

> This change breaks backward compatibility in `sign`. We cannot be sure that all Dapps will use the new sdk in the same time.
> 
> We should either handle `sign` URI with prefix and without prefix on wallets side or just don't add such prefix and assume that when there is no prefix we consume `sign` sdk.

Comment (jakubuid) [reply sign/pairing-uri.md:5]: Ahh sorry @Elyniss I didn't notice

File: sign/pairing-uri.md [hunk 2]
```diff
@@ -29,5 +29,20 @@
     relay = { protocol: "irn", data: "" }
 
 ```
-uri = "wc:7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=irn&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303"
+uri = "wc:sign-7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=irn&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303"
+```
+
+## Backwards Compatibility
+1. URI MAY not contain `sign-` prefix to bo valid
```
Comment (jakubuid) [inline sign/pairing-uri.md]: `bo` ?

Comment (Elyniss) [reply sign/pairing-uri.md]:
```suggestion
1. URI MAY not contain `sign-` prefix to be valid
```

**Review by jakobuid — APPROVED** [review]

File: sign/pairing-uri.md [hunk 3]
```diff
@@ -20,8 +20,11 @@
 ### Optional
 
 - `relay-data` (STRING) = hex data payload used for relay
+- `sdk` (STRING) = sdk that should handle this uri
 
-## Example
+## Examples
+
+1. Only required parameters
```
Comment (Talhaali00) [inline sign/pairing-uri.md]: maybe we call this the deprecated example since we want to move forward with the sdk parameter

Comment (Elyniss) [reply sign/pairing-uri.md]: Nice call, I like it

Comment (Elyniss) [reply sign/pairing-uri.md]: Also reordered them

**Comment by pedrouid** [comment]
There is an issue with this approach because a URI like Chat API uses will have a different schema than the one used by Auth and Sign so if you only add the SDK on the query param (which is at the end) then you can't properly parse the URI

**Comment by Elyniss** [comment]
Moved back to previous proposal without `sdk` parameter

File: auth/request-uri.md [hunk 4]
```diff
@@ -29,5 +29,5 @@
     relay = { protocol: “iridium”, data: “” }
 
 ```
-uri = “wc:7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=iridium&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303”
+uri = “wc:auth-7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=iridium&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303”
```
Comment (devceline) [inline auth/request-uri.md:32]: Can we change `iridium` to `irn` here please?

Comment (devceline) [inline auth/request-uri.md]: Can we change `iridium` to `irn` here?

Comment (llbartekll) [reply auth/request-uri.md]: good catch, I think it should be changed

File: auth/request-uri.md [hunk 5]
```diff
@@ -29,5 +29,5 @@
     relay = { protocol: “iridium”, data: “” }
```
Comment (jakubuid) [inline auth/request-uri.md]: And here as well `iridium` to `irn`

File: sign/pairing-uri.md [hunk 6]
```diff
@@ -29,5 +29,20 @@
...
+uri = "wc:sign-7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=irn&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303"
+```
+
+## Backwards Compatibility
+1. URI MAY not contain `sign-` prefix to be valid
+
+### Schema
+
+```
+    uri         = "wc" ":" topic [ "@" version ][ "?" parameters ]
```
Comment (bkrem) [inline sign/pairing-uri.md]: Leading whitespace:

```suggestion
uri         = "wc" ":" topic [ "@" version ][ "?" parameters ]
```

File: sign/pairing-uri.md [hunk 7]
```diff
@@ -29,5 +29,20 @@
...
+```
+
+### Example
+
+```
+    uri = "wc:7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=irn&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303"
```
Comment (bkrem) [inline sign/pairing-uri.md]: Leading whitespace (being rendered as dots in the markdown):

```suggestion
uri = "wc:7f6e504bfad60b485450578e05678ed3e8e8c4751d3c6160be17160d63ec90f9@2?relay-protocol=irn&symKey=587d5484ce2a2a6ee3ba1962fdd7e8588e06200c46823bd18fbd67def96ad303"
```

**Review by bkrem — APPROVED** [review]

**Comment by pedrouid** [comment]
@Elyniss is this good to merge?

**Pull request merged**